tools/appveyor.psm1

```diff
@@ -210,7 +210,8 @@ function Invoke-AppVeyorInstall
     catch { }
     if (!$haveLocalAccountTokenFilterPolicy)
     {
-        Write-Warning "LocalAccountTokenFilterPolicy not set.  Remoting tests will fail!"
+        Write-Warning "Setting the LocalAccountTokenFilterPolicy for remoting tests"
```

Comment:
@TravisEz13  Thanks, will change.  However the test is still failing in AppVeyor.  It looks like this is not the right location for creating the remoting account, maybe because it doesn't run or doesn't run in the same context/runspace as the test and the test is not getting the credential.  I am not familiar with AppVeyor ... is there a better place to be creating remoting test credentials?